README.md

```diff
@@ -12,6 +12,8 @@ Note: this project is under development and functionality will improve over time
 npm install react-native-file-download --save
 ```
 
+## Breaking change -- Now supports React Native v25+.  Not compatible with prior versions.
+
```

Comment:
I think we should also let people know that if you want to use this package along with react-native under v25 you should use react-native-file-download@0.0.9.